docker/images/jsonstor/Dockerfile

```diff
@@ -2,8 +2,7 @@
 FROM vertexproject/vtx-base-image:py311
 
 COPY synapse /build/synapse/synapse
-COPY setup.py /build/synapse/setup.py
-COPY MANIFEST.in /build/synapse/MANIFEST.in
+COPY pyproject.toml /build/synapse/pyproject.toml
```

Comment:
Any specific reason this didn't get the README copied in like the others?

Reply:
That is a mistake.